web_field_tooltip/README.rst

```diff
@@ -0,0 +1,99 @@
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:6a4fafacb03368529d861e303732dce4c182ee511090cb4fd9ca04d67347bae0
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/16.0/web_field_tooltip
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_field_tooltip
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/web&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module gives the possibility to add tooltips next to fields labels on any
+field of a model. The tooltip displays an html field.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to Settings / Technical / Database Structure / Fields Tooltips and
+  configure tooltips for a field for a model. You will then be able to see
+  on the form view of the model you chose, next to the label of the field you
+  selected, the tooltip appear.
+
+* You can also go directly on the form view of a model on which you want to
+  add a tooltip and use the Manage tooltips Action to add a new tooltip, or edit
+  or delete existing tooltips for this model.
+
+* For manager users, they can decide to display an helper to add a tooltip
+  on a field as some fields are not displayed with a label.
+
+Known issues / Roadmap
+======================
+
+* Find a way to update form views after tooltips update without refreshing manually
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us to smash it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/web/issues/new?body=module:%20web_field_tooltip%0Aversion:%2016.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* ACSONE SA/NV
+
+Contributors
+~~~~~~~~~~~~
+
+* Benoit Aimont <benoit.aimont@acsone.eu> (https://www.acsone.eu/)
```

Comment:
And Benjamin Willig, innit?

Reply:
![imbatman](https://github.com/OCA/web/assets/27724417/89bad68a-1269-4e46-a729-b20f66fd1e66)